guidelines/index.html

```diff
@@ -3004,13 +3004,13 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
                 <dt><dfn id="dfn-live" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">ライブ (live)</dfn></dt>
 <dd>
    
-   <p>現実の出来事から取り込まれ、放送遅延以上の遅延なく受け手に送信される情報。
+   <p>現実世界の出来事から取り込まれ、放送遅延を超える遅延なしに受信者に送信される情報。
    </p>
    
-   <div class="note" role="note" id="issue-container-generatedID-119"><div role="heading" class="note-title marker" id="h-note-119" aria-level="3"><span>注記 1</span></div><p class="">放送遅延は、短時間の (通常は自動的な) 遅れで、例えば放送局に放送のタイミング[queue→cue]の調整や音声 (又は映像) の検閲のための時間を与えるものだが、意味のある編集ができるほどのものではない。
+   <div class="note" role="note" id="issue-container-generatedID-119"><div role="heading" class="note-title marker" id="h-note-119" aria-level="3"><span>注記 1</span></div><p class="">放送遅延とは、短時間の (通常は自動的な) 遅延であり、例えば放送局に音声 (もしくは映像) 番組を頭出しする又は検閲する時間を与えるために使用されるものだが、大幅な編集を可能にするほど十分なものではない。
```

Comment:
「頭出しする又は検閲する時間を与える」は以下のいずれにもとれて、多義的になっています。

- (頭出しする) 又は (検閲する時間を与える)
- (頭出しする又は検閲する)時間を与える

あと cue → 頭出し feed → 番組 とするのは微妙にニュアンスが違うような気がしてしまいます。どちらも放送業界の専門用語としてそのままカタカナでも良いのではないかと思いましたが、どうでしょうか。

たとえばこう?

```suggestion
   <div class="note" role="note" id="issue-container-generatedID-119"><div role="heading" class="note-title marker" id="h-note-119" aria-level="3"><span>注記 1</span></div><p class="">放送遅延とは、短時間の (通常は自動的な) 遅延であり、例えば放送局に音声 (もしくは映像) フィードのキュー出しや検閲の時間を与えるために使用されるものだが、大幅な編集を可能にするほど十分なものではない。
```

Reply:
orは「又は」にする必要があるので…

```suggestion
   <div class="note" role="note" id="issue-container-generatedID-119"><div role="heading" class="note-title marker" id="h-note-119" aria-level="3"><span>注記 1</span></div><p class="">放送遅延とは、短時間の (通常は自動的な) 遅延であり、例えば放送局に音声 (もしくは映像) フィードのキュー出し又は検閲の時間を与えるために使用されるものだが、大幅な編集を可能にするほど十分なものではない。
```